tests/flow/test_effects.py

```diff
@@ -168,11 +168,12 @@ def test03_add_attribute_effect(self, expect_effect=True):
                 n.a = 1,
                 n.b = 'str',
                 n.c = True,
-                n.d = [1, [2], '3']
+                n.d = [1, [2], '3'],
+                n.v = vector32f([1.0, 2.0, 3.0])
```

Comment:
what about vector64?

Reply:
support for vector64 had been removed for the timbering